admin/autoupdater/files/usr/sbin/autoupdater

```diff
@@ -252,8 +257,10 @@ local function autoupdate(mirror)
   end
 
   if not autoupdater_version.newer_than(manifest.version, old_version) then
```

Comment:
i could change it, but as you said the autpupdater will be replaced - if you say : do it and i merge it , ok
otherwise, you could merge it already